src/validation.rs

```diff
@@ -1470,14 +1506,25 @@ fn validate_extension_modules(
 
     if is_linux {
         wanted.extend(GLOBAL_EXTENSIONS_POSIX);
-        wanted.extend(GLOBAL_EXTENSIONS_LINUX);
+        if python_major_minor == "3.13" {
+            wanted.remove("_crypt");
```

Comment:
Should these `wanted.remove` calls be encoded in constants too?

Reply:
Maybe, but notice we already split up the constants for LINUX but not POSIX and this is a much less invasive change. I'll look into addressing this separately, to avoid another CI run for something that's not blocking here.

Reply:
Adding a TODO for this since I need to push a commit anyway.